### PR TITLE
deps updated + moved deps into dev-dependencies

### PR DESCRIPTION
**Pull request by delight**
- minor: justify-left -> justify-start

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -125,19 +131,10 @@
...
-	"type": "module",
-	"dependencies": {
-		"@melt-ui/svelte": "^0.47.1",
-		"@unocss/preset-mini": "^0.55.7",
-		"@unocss/preset-wind": "^0.55.7",
-		"@unocss/reset": "^0.55.7",
-		"@unocss/transformer-directives": "^0.55.7",
-		"@unocss/transformer-variant-group": "^0.55.7",
-		"esm-env": "^1.0.0",
-		"unocss": "^0.55.7"
```
Comment (bennymi) [inline package.json:141]: I think let's leave `@melt-ui/svelte`, `@unocss/preset-wind`, `@unocss/transformer-directives`, and `unocss` as dependencies for now. The others we can safely add to devDependencies. And `esm-env` I think we can remove entirely if I'm not mistaken. I think it's from the previous version when I wasn't using Melt yet, and I also can't find it being used anywhere else, so please remove it.

File: src/docs/previews/accordion/colors/app.svelte [hunk 2]
```diff
@@ -1,59 +1,64 @@
...
-        icon: string;
-        titleBgStyle: string;
-    }
+	type ExampleItem = {
+		key: string;
+		title: string;
+		content: string;
+		icon: string;
+		titleBgStyle: string;
+	};
```
Comment (bennymi) [inline src/docs/previews/accordion/colors/app.svelte:10]: What changes were made here? Just spacing right?

**Review by bennymi — CHANGES_REQUESTED** [review]
Thanks for the PR. I think it's mostly good to merge, but I left some comments about the dependencies. I think we should keep the main ones as dependencies for now.

File: src/docs/previews/accordion/colors/app.svelte (hunk 2 above)
Comment (delight) [reply src/docs/previews/accordion/colors/app.svelte:10]: vscode is picking up the prettier settings and I have format on save activated

File: package.json (hunk 1 above)
Comment (delight) [reply package.json:141]: Ok I'll change that .. np

File: src/docs/previews/accordion/colors/app.svelte (hunk 2 above)
Comment (delight) [reply src/docs/previews/accordion/colors/app.svelte:10]: I had tried something some classes and prettier formated the file ... so if you like I can revert it

Comment (delight) [reply src/docs/previews/accordion/colors/app.svelte:10]: I had changed "justify-left" to "justify-start" in that file

Line 54:
				<div class="flex justify-start items-center gap-2">

Comment (bennymi) [reply src/docs/previews/accordion/colors/app.svelte:10]: @delight yeah no worries, looks good :)

**Review by bennymi — APPROVED** [review]

**Pull request merged**